WasatchNET/Spectrometer.cs

```diff
@@ -1957,9 +1957,14 @@ internal bool sendCmd(Opcodes opcode, ushort wValue = 0, ushort wIndex = 0, byte
                 wIndex,         // wIndex
                 wLength);       // wLength
 
-            bool expectedSuccessResult = true;
+            bool? expectedSuccessResult = true;
             if (isARM)
-                expectedSuccessResult = armInvertedRetvals.Contains(opcode);
+            {
+                if (opcode != Opcodes.SECOND_TIER_COMMAND)
+                    expectedSuccessResult = armInvertedRetvals.Contains(opcode);
+                else
+                    expectedSuccessResult = null; // no easy way to know, as we don't pass wValue as enum

```

Comment:
Perhaps we should add a new sendCmd2ndTier for this case. The old API had very few 2nd-tier "setters" (only getters), but Leigh's been adding a bunch of new ones.